frontend/components/TableContainer/DataTable/SoftwareNameCell/_styles.scss

```diff
@@ -14,12 +14,18 @@
 
   .software-name {
     @include ellipse-text();
-    max-width: 300px;
+    max-width: 150px;
+    @media (min-width: ($break-xs + 100px)) {
+      max-width: 200px;
+    }
+    @media (min-width: $break-sm) {
+      max-width: 250px;
+    }
```

Comment:
Here's an example where I didn't want to have to wait until 768px to increase the value. In this case, it's a one-time break point that's specific to this table, so need to create a new breakpoint. Instead, just add pixels to the existing breakpoint. 

This is an arguable approach. It would be cleaner to just wait until 768px. But I also think it's okay to dial in, we just don't want to set one-off values specific to that view as variables.

Reply:
I dont mind the one time breakpoints if we have first tried the other ones and found it doesn't work. Dont think we need to say we always can only ever used the breakpoints variables. The only thing I would add is we should comment in the code why we are using a one off.